### PR TITLE
Support for MicoAir743

### PR DESCRIPTION
**Pull request by Minderring**
Support for [MicoAir743](https://micoair.com/index.php/flightcontroller_micoair743/), a flight controller designed and produced by [MicoAir Tech.](https://micoair.com/).

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/MicoAir743/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+## UART Mapping
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (MAVLink2, DMA-enabled)
+ - SERIAL2 -> UART2 (DisplayPort, DMA-enabled)
+ - SERIAL3 -> UART3 (GPS, DMA-enabled)
+ - SERIAL4 -> UART4 (MAVLink2, DMA-enabled)
+ - SERIAL5 -> UART6 (RCIN, DMA-enabled)
+ - SERIAL6 -> UART7 (ESC Telemetry, DMA-enabled)
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/MicoAir743/README.md:34]:
```suggestion
 - SERIAL6 -> UART7 (TX only,ESC Telemetry, DMA-enabled)

```

File: libraries/AP_HAL_ChibiOS/hwdef/MicoAir743/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+ - SERIAL7 -> UART8 (DMA-enabled)
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/MicoAir743/README.md:35]:
```suggestion
 - SERIAL7 -> UART8 (User,DMA-enabled)

```

File: libraries/AP_HAL_ChibiOS/hwdef/MicoAir743/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+## RC Input
+
+The default RC input is configured on the UART6. The SBUS pin is inverted and connected to RX6. Non SBUS, single wire serial inputs can be directly tied to RX6 if SBUS pin is left unconnected. RC could  be applied instead at a different UART port such as UART1, UART4 or UART8, and set the protocol to receive RC data: `SERIALn_PROTOCOL=23` and change SERIAL5 _Protocol to something other than '23'.
+
+## OSD Support
+
+The MicoAir743 supports onboard OSD using OSD_TYPE 1 (MAX7456 driver).
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/MicoAir743/README.md:43]:
```suggestion
The MicoAir743 supports onboard OSD using OSD_TYPE 1 (MAX7456 driver). Simultaneously,  DisplayPort OSD is available on the HD VTX connector, See below.

```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
I few minor nits....this autopilot looks very good for small planes!
 I, myself, have two projects coming up for a tiny VTOL and a backpackable wing that this autopilot would be ideal for...when will it be available??

**Comment by Minderring** [comment]
> Just a few minor nits....this autopilot looks very good for small planes! I, myself, have two projects coming up for a tiny VTOL and a backpackable wing that this autopilot would be ideal for...when will it be available??

Thanks for your suggestion and MicoAir743 is coming soon.

**Comment by Hwurzburg** [comment]
let us know when you have completed testing and we can merge
I suggest you send a board to a few devs for support issues if they arise in the future

**Review by Hwurzburg — APPROVED** [review]

**Comment by Minderring** [comment]
> let us know when you have completed testing and we can merge I suggest you send a board to a few devs for support issues if they arise in the future

Okay. I will do it in few days.

**Comment by Minderring** [comment]
I have completed testing and it works well.

**Pull request merged**